src/main/java/constants/DOW.java

```diff
@@ -0,0 +1,43 @@
+package constants;
+
+public enum DOW {
+    MONDAY,
+    TUESDAY,
+    WEDNESDAY,
+    THURSDAY,
+    FRIDAY,
+    SATURDAY,
+    SUNDAY;
+
+    public static DOW getEnum(String type) {
```

Comment:
I like how you implemented this! Better than the solution I came up with in my branch. (May steal this)